smmips/smmips.py

```diff
@@ -147,7 +431,12 @@ def main():
                           help = 'Cut-off value for the length of the de-gapped overlap between read1 and read2. Default is 60bp')
     a_parser.add_argument('-mt', '--Matches_threshold', dest='matches_threshold', type=float, default=0.7, \
                           help = 'Cut-off value for the number of matching positions within the de-gapped overlap between read1 and read2. Used only if report is True. Default is 0.7')
-        
+    
+    # merge chromosome-level files
+    m_parser = subparsers.add_parser('merge', help='Merges all the chromosome-level stats and alignment files')
```

Comment:
i can't see here how the tool is getting a list of files to merge.  are they all expected to be in a single directory, and where is that indicated?  perhaps each extract command is run with the same output folder?
its likely the implementation in Niassa will result in the data from each chromosome ending up in a distinct location

Reply:
It is merging files located in an output directory. I'm not sure if this is going to be an issue or not with the workflow. 
I'm suggesting to keep that logic in the python tool for now and adjust as needed if this creates issues with Niassa.

Reply:
yes, as discussed the approach would be to have a task in the workflow that pulls all the data from the per-chromosome run into a single directory.  then run the merging data on that folder